backend/src/main/java/saci/domain/service/AlreadyExistsException.java

```diff
@@ -0,0 +1,8 @@
+package saci.domain.service;
```

Comment:
Doesn't this class already exist? 
If yes, please use the already created one instead of duplicating this one.